packages/cli/src/commands/introspect.ts

```diff
@@ -17,7 +18,9 @@ export async function introspect(
   const renderer = options.renderer || new ConsoleRenderer();
 
   try {
-    const schema = await loadSchema(schemaPointer);
+    const schema = await loadSchema(schemaPointer, {
+      token: options.token,
```

Comment:
Why are you passing it as `token`?  @kamilkisiela 
I think in case of url, you need to specify `{ headers: { Authorization: options.token }}`

Reply:
Yeah but `loadSchema` is not from `graphql-toolkit`, it's a wrapper and `token` is to pass Github Token so we can load schema from github repo.

Reply:
missed that 😄